test/Java8andUp/build.xml

```diff
@@ -149,28 +151,59 @@
+						<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+							<src path="${src}" />
+							<src path="${src_90}" />
+							<src path="${src_90_current}" />
+							<src path="${TestUtilities}" />
+							<src path="${transformerListener}" />
+							<compilerarg line='${addExports}' />
+							<compilerarg line='${addModules}' />
+							<!-- PR117298-->
+							<exclude name="**/Cmvc194280.java" />
+							<exclude name="**/resources/**"/>
+							<!-- requires special compilation methods -->
+							<exclude name="**/NoSuchMethod/**" />
+							<classpath>
+								<pathelement location="../TestConfig/lib/asm-all.jar" />
+								<pathelement location="../TestConfig/lib/testng.jar" />
+								<pathelement location="../TestConfig/lib/jcommander.jar" />
+								<pathelement location="../TestConfig/lib/commons-cli.jar" />
+								<pathelement location="../TestConfig/lib/javassist.jar" />
+							</classpath>
+						</javac>
+					</then>
+					<else>
+						<property name="addModules" value="--add-modules java.se.ee,openj9.sharedclasses" />
+						<property name="addExports" value="--add-exports jdk.management.agent/jdk.internal.agent=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.oti.vm=ALL-UNNAMED --add-exports java.base/com.ibm.oti.util=ALL-UNNAMED --add-exports java.base/jdk.internal.vm.annotation=ALL-UNNAMED --add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/com.ibm.jit.crypto=ALL-UNNAMED --add-exports java.base/com.ibm.jit=ALL-UNNAMED --add-exports java.base/com.ibm.oti.reflect=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind.annotation=ALL-UNNAMED --add-exports java.xml.bind/javax.xml.bind=ALL-UNNAMED" />
+						<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+							<src path="${src}" />
+							<src path="${src_90}" />
+							<src path="${src_90_latest}" />
+							<src path="${TestUtilities}" />
+							<src path="${transformerListener}" />
+							<compilerarg line='${addExports}' />
+							<compilerarg line='${addModules}' />
+							<!-- PR117298-->
+							<exclude name="**/Cmvc194280.java" />
+							<exclude name="**/resources/**"/>
+							<!-- requires special compilation methods -->
+							<exclude name="**/NoSuchMethod/**" />
+							<classpath>
+								<pathelement location="../TestConfig/lib/asm-all.jar" />
+								<pathelement location="../TestConfig/lib/testng.jar" />
+								<pathelement location="../TestConfig/lib/jcommander.jar" />
+								<pathelement location="../TestConfig/lib/commons-cli.jar" />
+								<pathelement location="../TestConfig/lib/javassist.jar" />
+							</classpath>
+						</javac>
+					</else>
+				</if>
```

Comment:
Instead of using if else, could we use variables to avoid code duplication? The differences between if and else are the following:
 - `${src_90_current}` and `${src_90_latest}`
 -  `--add-exports java.management/sun.management=ALL-UNNAMED` and `--add-exports jdk.management.agent/jdk.internal.agent=ALL-UNNAMED`

Reply:
good idea. I just pushed this update